homeassistant/components/tuya/sensor.py

```diff
@@ -742,6 +742,35 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             icon="mdi:progress-clock",
         ),
     ),
+    # Humidifier
+    # https://developer.tuya.com/en/docs/iot/s?id=K9gf48qwjz0i3
+    "jsq": (
+        TuyaSensorEntityDescription(
+            key=DPCode.HUMIDITY_CURRENT,
+            name="Humidity",
+            device_class=SensorDeviceClass.HUMIDITY,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.TEMP_CURRENT,
+            name="Temperature",
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.TEMP_CURRENT_F,
+            name="Temperature Fahrenheit",
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+            entity_registry_enabled_default=False,
```

Comment:
Why disable this one by default?

Reply:
I thought of keeping only one of the units enabled, if the user prefers the other, he will activate it (both in the sensor and in the number...) just so as not to have two active controls with the same function.

Better to leave everything enabled?

Reply:
They don't have to be available both. I have devices that have either (not both).

Reply:
I really got confused by this. I changed...